cmake/Findabsl.cmake

```diff
@@ -2,6 +2,14 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+find_package(absl CONFIG)
+
+if(absl_FOUND)
+  return()
+endif()
+
+message("Abseil not found via CMake. Probably not a Conan build. Using the copy from third_party/")
```

Comment:
This find_package call ensures we are not linking two versions of Abseil into the same build.
(It was an oversight from the previous introduction of this find script and became a problem now due to the Abseil update.)